src/test/java/org/example/tests/CheckoutTest.java

```diff
@@ -1,35 +1,41 @@
 package org.example.tests;
 
-import org.example.enums.UserCredentials;
-import org.example.pages.*;
 import org.example.utils.BaseTest;
 import org.testng.Assert;
 import org.testng.annotations.Test;
 
 import java.util.Arrays;
 import java.util.List;
 
-public class CheckoutFlowTest extends BaseTest {
+public class CheckoutTest extends BaseTest {
 
     @Test
     public void testCompleteCheckoutWithMultipleItems() {
 
         List<String> products = Arrays.asList("Sauce Labs Backpack", "Sauce Labs Bike Light");
 
-        getLoginPage().login(UserCredentials.STANDARD_USER);
+        loginToAccount();
         Assert.assertTrue(getProductPage().isProductListVisible(), "Product page not loaded.");
 
 
         getProductPage().addProductsToCart(products);
         Assert.assertEquals(getProductPage().getCartCount(), products.size(), "Incorrect cart badge count.");
         getProductPage().clickCartIcon();
```

Comment:
use         ProductPageBase productPage = loginToAccount();
it allows to reuse productPage